test/test_python_dispatch.py

```diff
@@ -522,5 +522,12 @@ def wrap(e):
         with self.assertRaisesRegex(RuntimeError, "but got None"):
             out.backward()
 
+    def test_storage_can_be_converted_to_python_object(self):
+        with enable_python_mode(LoggingTensor):
+            s = torch.Storage()
+            z = LoggingTensor(torch.empty([]))
```

Comment:
Does z needs to be a LoggingTensor here since we are using a mode?

Reply:
Technically, as of today, the test would also work with a plain `z = torch.empty([])` statement. However, by design `enable_python_mode()` is meant to be used along with tensors of the type specified as its argument. So just wanted to make sure I follow that.

Reply:
Ok!